daemon/algod/api/algod.oas2.json

```diff
@@ -3972,6 +4045,74 @@
           }
         }
       }
+    },
+    "SimulationUnnamedGroupResources": {
+      "description": "If unnamed resource access is allowed, this is the subset of unnamed resources that were accessed by this group and could benefit from group resource sharing. In other words, there are no restrictions on where these resources must be placed in the transaction group. Also see property 'unnamed-resources-accessed' in SimulateTransactionResult to see unnamed resources accessed by each transaction which cannot benefit from group sharing; in contrast, these resources must be placed local to that transaction.",
+      "type": "object",
+      "required": [
+        "resources"
+      ],
+      "properties": {
+        "resources": {
+          "$ref": "#/definitions/SimulationUnnamedResourceAssignment"
+        },
+        "asset-holdings": {
+          "description": "The unnamed asset holdings that were referenced. The order of this array is arbitrary.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/AssetHoldingReference"
+          }
+        },
+        "app-locals": {
+          "description": "The unnamed app local states that were referenced. The order of this array is arbitrary.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ApplicationLocalReference"
+          }
+        }
+      }
+    },
+    "SimulationUnnamedResourceAssignment": {
```

Comment:
I suppose I'd advocate "Simulation" -> "Simulated" for all these result types.  And "Assignment" -> "Access"

Reply:
Renamed to "SimulateUnnamedResourcesAccessed" in 9b112ee23. I see https://github.com/algorand/go-algorand/pull/5366#discussion_r1279362615 has further comments, but I have not yet considered those

For now, I've opted to keep the prefix as "Simulate" instead of "Simulated" for consistency with the rest of the related structs; this struct is a field of "SimulateTransactionResult", which is in "SimulateTransactionGroupResult" which is in "SimulateResponse"

Reply:
Though unrelated, this makes me slightly alert about my previous naming on exec trace and even earlier on eval override: There are 3 objects start with `simulation` rather than `simulate`:
1. `SimulationEvalOverrides`
2. `SimulationOpcodeTraceUnit`
3. `SimulationTransactionExecTrace`

do we want to make 2 and 3 align with the decision above? (point 1 has been released a while ago)

Reply:
IMO I don't think it's worth the effort to change any names that have already been released. Going forward we can stick to whatever convention we decide

Reply:
That's fair.